### ISSUE TITLE
This extension may soon no longer be supported because it doesn't follow best practices for Chrome extensions.

### ISSUE THREAD

**Issue by terpmeister** [browserpass/browserpass-extension#354]
### Warning from Google Chrome Extensions

Anybody seeing this? 

Firefox is still working OK. In my Dev version of Chrome it outright prevent me from running it anymore. Regular chrome at the moment is letting me still run it for the time being.

**Comment by maximbaz** [comment]
Without knowing for sure, it might be caused by #320. cc @erayd do you know more about this?

**Comment by erayd** [comment]
@maximbaz That is precisely what it is. I had entirely forgotten that deadline was coming up!

For the moment, Chrome / Google will just moan about it, but will continue to let you use it. But we do need to get it migrated.

**Comment by CarloWood** [comment]
30 Dec 2024 now - and I am still seeing this.
Does this mean I can't use browserpass (and therefore can't use pass?!) I just #&$&%@ installed it! :(

**Comment by maximbaz** [comment]
It works so far - we are tracking progress in https://github.com/browserpass/browserpass-extension/issues/320, feel free to subscribe. Help is also very welcome, just keep discussions open as someone is already working on this, to avoid duplicate work.

**Comment by CarloWood** [comment]
Talking about coincidences... about the same time I posted the above https://github.com/patgmiller/browserpass-extension.git did commit a "first attempt" at fixing this on the branch 320-migrate-v3-manifest - which for some reason I found 8 minutes after he committed it.  I tested that extension and it works, although it gives me this error:
https://gyazo.com/49a79db443a32c46d35d82b38b3e6dce

**Comment by jounisuo** [comment]
<img width="405" alt="Image" src="https://github.com/user-attachments/assets/a28ad06b-e6ca-4e28-b50d-37f058f1e08f" />
This day has come

**Comment by leroyz84** [comment]
> <img alt="Image" width="405" src="https://private-user-images.githubusercontent.com/12610613/418468248-a28ad06b-e6ca-4e28-b50d-37f058f1e08f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDA5ODc4MjEsIm5iZiI6MTc0MDk4NzUyMSwicGF0aCI6Ii8xMjYxMDYxMy80MTg0NjgyNDgtYTI4YWQwNmItZTZjYS00ZTI4LWI1MGQtMzdmMDU4ZjFlMDhmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzAzVDA3Mzg0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmZjU2ZWUyOTdlOGQ4NGE5ZGI2MDQzMDQxYWE0MDBkZTk2YTQ2ZjJkNjMyNmQ1ZDJiM2UxOWM0NWI1MTY3YzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.HFErZRMF6cdpZqaoD_Fxp7aKtlQK9DsKNc6gAg_QISU"> This day has come

Which version of Chrome are you using? (133.0.6943.141 is still usable)

**Comment by jounisuo** [comment]
I am on 133.0.6943.142

**Comment by computator** [comment]
I had that happen a week or two ago, but you can click the little toggle to turn it back on anyways after clicking through a warning.

**Comment by jounisuo** [comment]
> I had that happen a week or two ago, but you can click the little toggle to turn it back on anyways after clicking through a warning.

Ah, right you are!

**Comment by mhemeryck** [comment]
Same here, can not turn it on again. Small side note: I'm on a nix-based system; chrome indicates "my browser is managed by an organization".

**Comment by computator** [comment]
You can't enable them from that warning pop-up. For me I had to go to the manage extensions page and then enable the extension with the toggle in the bottom right of the extension card. I believe that worked for me on stable too, but I do run Chrome Dev on the majority of my computers so that could be a factor.

**Comment by mhemeryck** [comment]
No, I really can't turn the extension back on again; the toggle is greyed out.

![Image](https://github.com/user-attachments/assets/a676462d-3f50-4f84-a852-87579ec986f3)

**Comment by louwers** [comment]
@mhemeryck You can. Just go to chrome://extensions click ... and then Keep for Now.

**Comment by mhemeryck** [comment]
> [@mhemeryck](https://github.com/mhemeryck) You can. Just go to chrome://extensions click ... and then Keep for Now.

Not working for me ... but I think it more has to do with my browser setup.

**Comment by maximbaz** [comment]
v3.10.0 landed in webstore and no longer has any deprecation warnings 🚀
